context-monitoring/src/main/java/de/atb/context/monitoring/monitors/file/FilePairSystemMonitor.java

```diff
@@ -17,6 +17,7 @@
 import java.io.File;
 import java.io.FilenameFilter;
 import java.io.IOException;
+import java.nio.file.*;
```

Comment:
nit: try to avoid wildcard imports

Comment also applies to the other two file monitor classes.

Reply:
done with the "organise imports" commit